gateway/project.go

```diff
@@ -39,7 +39,7 @@ func (g *Gateway) GetProject(ctx context.Context, projectId string) (*entity.Pro
 		Project *entity.Project `json:"projectById"`
 	}
 	if err := g.gqlClient.Run(ctx, gqlReq, &resp); err != nil {
-		return nil, err
+		return nil, errors.ProjectNotFound
```

Comment:
Any reason we're masking the error from users?

Additionally, it would be really good to have error reporting in the CLI